charts/grnoc-telegraf/values.yaml

```diff
@@ -9,6 +9,9 @@ Instance: ""
 # Optionally write metrics to stdout and local file in container.
 writeToStdout: true
 
+# Specifies the amount of memory (in GB) to request
+requestMemory: "1000"
```

Comment:
Looks like this value is requesting 1000GB according to the comment above, is that accurate?

Reply:
ooopsies, changed that to "1G"